### PR TITLE
Backport from master

### PR DESCRIPTION
**Pull request by petedyerarm**
Take the changes from the master branch back into the dev branch.

### CODE REVIEW

File: maestro/deb/build.sh [hunk 1]
```diff
@@ -4,10 +4,8 @@
 PELION_PACKAGE_NAME="maestro"
 PELION_PACKAGE_DIR=$(cd "`dirname \"$0\"`" && pwd)
 
-PELION_PACKAGE_PRE_BUILD_CALLBACK='select_python 2'
-
 declare -A PELION_PACKAGE_COMPONENTS=(
-    ["https://github.com/armPelionEdge/maestro.git"]="d6f1e29c2994c34adcf544d0fff50fd31fd01cdb")
+    ["https://github.com/armPelionEdge/maestro.git"]="718a61ce09dabc60075322e1c255c345aa7927e6")
```
Comment (JanneKiiskila) [inline maestro/deb/build.sh:8]: Domain is wrong, needs to be updated to PelionIoT.

File: kubelet/deb/build.sh [hunk 2]
```diff
@@ -5,22 +5,8 @@ PELION_PACKAGE_NAME="kubelet"
 PELION_PACKAGE_DIR=$(cd "`dirname \"$0\"`" && pwd)
 
 declare -A PELION_PACKAGE_COMPONENTS=(
-    ["https://github.com/PelionIoT/edge-kubelet.git"]="v1.0.0")
+    ["https://github.com/armPelionEdge/edge-kubelet.git"]="v1.1.0")
```
Comment (JanneKiiskila) [inline kubelet/deb/build.sh:8]: Domain is wrong, needs to be updated to PelionIoT.

File: maestro/deb/build.sh (hunk 1 above)
Comment (petedyerarm) [reply maestro/deb/build.sh:8]: I'd prefer to leave this PR for taking the changes from master => dev.
There are 38 places in the dev branch where the domain name needs to be updated, so those can be done in separate PR.

File: kubelet/deb/build.sh (hunk 2 above)
Comment (petedyerarm) [reply kubelet/deb/build.sh:8]: I'd prefer to leave this PR for taking the changes from master => dev.
There are 38 places in the dev branch where the domain name needs to be updated, so those can be done in separate PR.

**Review by JanneKiiskila — APPROVED** [review]

File: maestro/deb/build.sh (hunk 1 above)
Comment (JanneKiiskila) [reply maestro/deb/build.sh:8]: Roger that.

**Pull request merged**